### ISSUE TITLE
alert sound options

### ISSUE THREAD

**Issue by gempickfordwaugh** [timomer/HAPP#40]
Is this where I leave a feature request?  There currently isn't an option to change the alert sound options.  If you aren't using a watch it's not easy to get woken up to a proposed basal change, so could be a helpful feature?

**Comment by timomer** [comment]
Yep, this is the place!

Im thinking settings...
Enable audio TBR alerts - Y/N
Night Only - Y/N
Night Time Period - HH:MM until HH:MM

Let me know if you have any suggestions?

**Comment by gempickfordwaugh** [comment]
To me, it would make most sense to just replicate the settings of xdrip/xdrip+ alerts as that's an interface people are used to.  So: Alert Tone, Select time for alert: all day / HH:MM until HH:MM, Override phone Silent Mode, Vibrate on alert, and Disable alert.

But I don't know what that means in terms of programming so go with an easier option if necessary!

**Comment by Rocklita** [comment]
I would LOVE the ability to choose my own audible alert with variable settings. Using xDrip+ right now and can't seem to get them to work. It'll vibrate but no audio.

**Comment by iversenshotrods** [comment]
Same here, vibrate only and no audio if and when I get alerts (I am missing alerts also)

**Comment by timomer** [comment]
Yep thats correct, there are no audio alerts right now just vibrate. @iversenshotrods ill read over your issue later today when I have a few spare mins, thanks

**Comment by davidmlb** [comment]
Dear Tinomer, is the alarm sound solved? I'm only receiving the vibration mode

Thanks

**Comment by timomer** [comment]
Hi,

HAPP only support vibration for alerts, I have added sounds as a feature request for HAPP+. Unfortunately, I have been very busy this summer so HAPP+ dev has been on hold.

Due to that, I have added a simple option to add notification sound to HAPP Temp Basal notifications, see the option in Notification Prefs - note that this will play the default system notification sound.

For HAPP+ I will make this much more feature rich. 

Please test and report back.
https://drive.google.com/file/d/0BxE8lMx4AjLiNmdieGxmbC1HLU0/view?usp=sharing

**Comment by davidmlb** [comment]
Thanks, I downloaded it and then tried to  install, but appears a message the application was not installed.

Rgards

**Comment by timomer** [comment]
Hey, I just downloaded and installed the apk ok, could it be possible that you are using a build of HAPP that you or someone else compiled?

Export your HAPP settings, uninstall and try this the new apk and see if that works?
